Workflow failure: [PR #759] - run-acceptance-tests

## Workflow Failure

Triggered by PR: https://github.com/pulumi/pulumi-ec/pull/759

[run-acceptance-tests](https://github.com/pulumi/pulumi-ec/blob/master/.github/workflows/run-acceptance-tests.yml) has failed. See the list of failures below:

- [2026-04-08T04:19:07.000Z](https://github.com/pulumi/pulumi-ec/actions/runs/24117040064)
  - ❌ [lint / lint](https://github.com/pulumi/pulumi-ec/actions/runs/24117040064/job/70364237325): lint (failure)

Error ([link](https://github.com/pulumi/pulumi-ec/actions/runs/24703479526/job/72252313806)):

```
Error: can't load config: the Go language version (go1.25) used to build golangci-lint is lower than the targeted Go version (1.26.1)
The command is terminated due to an error: can't load config: the Go language version (go1.25) used to build golangci-lint is lower than the targeted Go version (1.26.1)
make: *** [Makefile:228: lint] Error 3
```

golangci-lint Go version mismatch (go1.25 vs targeted 1.26.1). Fix: [https://github.com/pulumi/ci-mgmt/pull/2130](https://github.com/pulumi/ci-mgmt/pull/2130)

--- a/mise.toml
+++ b/mise.toml
@@ -2,4 +2,3 @@
 "vfox-pulumi:pulumi/pulumi-std" = "2.2.0"
 "vfox-pulumi:pulumi/pulumi-converter-terraform" = "1.2.4"
 "vfox-pulumi:pulumi/pulumi-random" = "4.18.5"
-golangci-lint = "2.7.0"

